### PR TITLE
copy lambda-ms-teams-notifier

### PR DESCRIPTION
**Pull request by maxhaensel**

### CODE REVIEW

**Review by PhilippSchmidCoC — DISMISSED** [review]

File: aws/lambda-ms-teams-notifier/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+This function is just a notifier which pushes Cloudwatch Alarms messages (triggered via SNS) to MS Teams by using the channel's hook url.
+    
+## Used Services & Features
+* AWS Lambda including IAM Role and Policy
+* Python 3.8 (no external libs used)
+
+## Usage
+```
+module "ms-team-notifier-lambda" {
+    source = "git@github.com:codecampn/terraform-modules-aws-lambda-ms-teams-notifier.git"
```
Comment (norchen) [inline aws/lambda-ms-teams-notifier/README.md]: Wrong implementation link is shown. How will the source be included in future?

**Review by norchen — COMMENTED** [review]
Module Source including is clear? (see line comment)

File: aws/lambda-ms-teams-notifier/README.md (hunk 1 above)
Comment (norchen) [reply aws/lambda-ms-teams-notifier/README.md]: solved with current commit

**Review by norchen — APPROVED** [review]
lgtm 🥂

**Pull request merged**